src/ui_ng/lib/src/create-edit-rule/create-edit-rule.component.ts

```diff
@@ -144,7 +144,9 @@ export class CreateEditRuleComponent implements AfterViewChecked {
   }
   get connectAbled():boolean{
     return !this.createEditRule.endpointId &&  !this.isCreateEndpoint;
-
+  }
+  get hasConnectData():boolean{
```

Comment:
The name of method is not matched your expression.

has, but you use "!" that means not existing